reports.py

```diff
@@ -686,27 +705,60 @@ def plot_coverage(
         # Set the tick labels font
         for label in (ax.get_xticklabels() + ax.get_yticklabels()):
             label.set_fontsize(font_size)
-
+            
+        # Binning
+        bin_size = 1
+        if bin_large_plots:
+            inner_plot_width_inches = ax.get_window_extent().transformed(fig.dpi_scale_trans.inverted()).width
+            inner_plot_width_px = inner_plot_width_inches * fig.dpi # width of actual plot (sans whitespace and y axis text)
+            bins_per_pixel = 1 # increase to make smaller (but less visible) bins
+            
+            # Parse summary statistic
+            if(binning_summary_statistic.lower() == "min" or binning_summary_statistic.lower() == "minimum"):
```

Comment:
Constraining the param above with valid `choices` would simplify this a bit

Reply:
Agreed.